src/Azure.Functions.Cli/Helpers/DockerHelpers.cs

```diff
@@ -62,6 +62,7 @@ internal static async Task<TriggersPayload> GetTriggersFromDockerImage(string im
             var containerId = string.Empty;
             try
             {
+                // TODO: This could be a single Docker run command
```

Comment:
I used to have issues with sharing volumes from a Windows machine to a container that would work with all configs for Docker for Windows, so used `docker cp` instead.